average.js

```diff
@@ -1,6 +1,13 @@
-
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  let count = 0;
```

Comment:
Functionality is translated to working code nicely +1
We name variables, so that readers find it easy to see their intent without reading all of the code.
Try giving a better name than `count`, so readers will direcly get its intent.